package-lock.json

```diff
@@ -1,950 +1,8 @@
 {
   "name": "js-explorer",
-  "version": "1.0.1",
-  "lockfileVersion": 2,
+  "version": "1.1.7",
+  "lockfileVersion": 1,
```

Comment:
I don't know, but maybe downgrading the lockfileVersion and triggering 971 lines of changes in the lock is not that great idea?

Reply:
Strange, it was updated after I ran `npm install`

Reply:
maybe the commit with the package-lock is not required for the tweaks you are proposing?